### ISSUE TITLE
why use attention smoothing?

### ISSUE THREAD

**Issue by xinqipony** [mozilla/TTS#49]
I saw this:
 alignment  = torch.sigmoid(alignment) / torch.sigmoid(alignment).sum(dim=1).unsqueeze(1)

any experiment on this, is it improving anything?

**Comment by xinqipony** [comment]
so, multiply attention values by 5 , is to do attention sharpen?

**Comment by erogol** [comment]
@xinqipony yes. Using sigmoid sharpens enough for training and ultiplying by 5 sharpens more for inference. However, I suggest this assuming you use TTS.
